README.md

```diff
@@ -72,3 +72,6 @@ The following options are available.
  * `--lsm_port` port to use to ssh to the remote orchestrator, overrides INMANTA_LSM_PORT
  * `--lsm_environment` the environment to use on the remote server (it is created if it doesn't exist), overrides INMANTA_LSM_ENVIRONMENT
  * `--lsm_noclean` Don't cleanup the orchestrator after last test (for debugging purposes)
+ * `--ssl` Use SSL
+ * `--token` The token to use when using SSL
+ * `--ca_cert` The certificate to use when performing authentication
```

Comment:
We prefix all the argument related to this pytest plugin with `lsm_`, could you add it for those new arguments as well?  (Same for the environment variables, we use `INMANTA_LSM_...`)

Reply:
@Hugo-Inmanta The rationale behind this prefixing is the following: `pytest-inmanta-lsm` is a pytest plugin. This means that these config options will be available for use in the `pytest` command. It's important that we don't get a config between the config options we define in this plugin and the config options defined by other pytest plugins. As such we prefix all config option with `lsm_`, that highly reduce the change of a naming conflict.